mk/xamarin.mk

```diff
@@ -1,6 +1,6 @@
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := cebe977dc69e3ed628f3c87e349d971f52dd4b34
-NEEDED_MACCORE_BRANCH := master
+NEEDED_MACCORE_VERSION := 874818b0df27469298fcdb4b5fed61346c57f8b9
+NEEDED_MACCORE_BRANCH := mono-2017-02
```

Comment:
Any maccore changes should be merged to maccore/master first (assuming xamarin-macios/master works with those changes).